sdk/rust/src/sqlite.rs

```diff
@@ -0,0 +1,112 @@
+wit_bindgen_rust::import!("../../wit/ephemeral/sqlite.wit");
+
+use sqlite::Connection as RawConnection;
+
+/// Errors which may be raised by the methods of `Store`
+pub type Error = sqlite::Error;
+
+/// A parameter used when executing a sqlite statement
+pub type DataTypeParam<'a> = sqlite::ValueParam<'a>;
```

Comment:
suggestion: `ParameterValue` would be consistent with our other Rust RDBMS SDKs, and is clear that it's a _value_ rather than a data type.  And similar comment for `DataTypeResult` - the other SDKs use `DbValue` for this.